src/main/scala/sbtlicensereport/SbtLicenseReport.scala

```diff
@@ -12,18 +12,18 @@ object SbtLicenseReport extends AutoPlugin {
   object autoImportImpl {
     // Types and objects to auto-expose
     type LicenseCategory = sbtlicensereport.license.LicenseCategory
-    def LicenseCategory = sbtlicensereport.license.LicenseCategory
+    val LicenseCategory = sbtlicensereport.license.LicenseCategory
```

Comment:
I guess in theory `def` could potentially prevent eager evaluation and memory leaks, but likely a moot here since most of these are used in the setting code below?

Reply:
Yes correct